check_borg

```diff
@@ -58,6 +60,11 @@ while getopts ":vhR:a:c:w:" opt; do
 		R)
 			export "BORG_REPO=$OPTARG"
 			;;
+		C)
+			[ -e "${OPTARG}" ] || error "Configuration file '${OPTARG}' does not exist."
+			[ -r "${OPTARG}" ] || error "Could not read configuration file '${OPTARG}'."
+			source "${OPTARG}" || error "Could not source configuration file '${OPTARG}'."
```

Comment:
`source` doesn't do the trick, but `.` does. ;-)

Reply:
@z3dm4n Why this? `source` and `.` should be equivalent!?

Reply:
`source` is a bash shell (but not bash-only, e.g. zsh uses it too) built-in command. See `type source`. The `.` command is the (POSIX) bourne shell equivalent to the bash `source` command. Because your script uses /bin/sh, `source` doesn't work. /bin/sh should be a link to a bourne-compatible POSIX shell like dash (on Debian, Ubuntu) or ash on *BSD. In bash both commands are essentially equivalent.

Reply:
Thanks for clarifying this. When `/bin/sh -> /bin/bash`, you can't really determine, which is a POSIX and which is a bash feature. I've [read up the POSIX manual for `dot`](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#dot) and now I've got a misconception less. Thanks for noting this.